src/scales/quantitativeScale.ts

```diff
@@ -23,7 +25,26 @@ module Plottable {
     }
 
     protected _getExtent(): D[] {
-      return this._domainer.computeDomain(this._getAllExtents(), this);
+      var computedExtent = this._domainer.computeDomain(this._getAllExtents(), this);
+      var e0 = computedExtent[0];
+      var e1 = computedExtent[1];
+      if (this._autoMin != null) {
+        if (e0 < this._autoMin) {
+          e0 = this._autoMin;
+        }
+        if (e1 < this._autoMin) {
+          e1 = this._autoMin;
+        }
+      }
+      if (this._autoMax != null) {
+        if (e0 > this._autoMax) {
+          e0 = this._autoMax;
+        }
+        if (e1 > this._autoMax) {
+          e1 = this._autoMax;
+        }
```

Comment:
`Math.min` only works on `number`s, so I am using `<` and `>` to have it work on `number`-like objects.